Sorstr.py

```diff
@@ -1,55 +0,0 @@
-from glob import glob
-
-import re
-
-
-class Sorstr:
-    inverted_index = {}
-
-    def index(self, pattern: str) -> None:
-        """
-        Index files indicated by the pattern.
-
-        :param pattern: The pattern to use to look for files 
-        """
-        self.inverted_index = {}
-
-        files = glob(pattern)
-        files.sort()
-        for file in files:
```

Comment:
Using `document` as variable name might also enhance legibility, because the use of this word is consistent with how people talk about search.